azerothcore/azerothcore-wotlk
# fix(Core/BWL): Nefarian weekly adds spawn mechanics

## Changes Proposed:
-  Now Nefarius spawn only 2 colours of Drakonids per instance.
-  The add spawn mechanic is handled by its proper npc and not by Nefarius.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10952

## SOURCE:


## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. Go to BWL
2. Talk with Nefarius
3. You should see only 2 types of Drakonids (and some Chromatics).
4. Try to shutdown the worldserver, enter the instance again and start the fight, the colours should be the same as before.
5. On a clean save, it should change (or not, depending your luck :P).
6. Try wiping on different scenarios to see if the fight restarts fine.
